bravado/client.py

```diff
@@ -215,6 +215,16 @@ def construct_params(self, request, op_kwargs):
             if not remaining_param.required and remaining_param.has_default():
                 marshal_param(remaining_param, None, request)
 
+    def log_warning_for_deprecated_op(self):
+        # TODO: Move this logic to bravado_decorators
```

Comment:
I think this is actually fine remaining in bravado.  Treating `x-deprecated-date` and `x-removal-date